### ISSUE TITLE
Bug : Blue color spot on agenda 3

### ISSUE THREAD

**Issue by 4SH-gaupee** [haydenull/logseq-plugin-agenda#308]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

Blue spot in agenda 3, this is also visible in documentation screenshot.

### Reproduce the Bug

Open agenda 3

### Screenshots

![image](https://github.com/haydenull/logseq-plugin-agenda/assets/144819135/0884f984-dfc2-46e0-b827-6851d2ea9cbf)

### Are you willing to submit a PR? If you know how to fix the bug.

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by haydenull** [comment]
‌‌‌‌‌Um, I feel that a plain background is too monotonous so I intentionally added some color.

Do you think this will interfere with reading?

**Comment by 4SH-gaupee** [comment]
I agree with the plain background but the blue stain is not really good looking and is distracting, maybe a fade from white to light gray across the window could look better ? But I'm not a UI/UX designer
